Crashteroids Starter/Assets/RW/Scripts/Laser.cs

```diff
@@ -50,10 +56,24 @@ private void OnCollisionEnter(Collision collision)
     {
         if (collision.gameObject.GetComponent<Asteroid>() != null)
         {
-            spawner.SpawnShrapnel(collision.gameObject.transform.position, collision.collider);
-            Game.AsteroidDestroyed();
+           if(collision.gameObject.GetComponent<Asteroid>().health < 1)
+            {
+                Game.AsteroidDestroyed();
+                Destroy(gameObject);
+                spawner.asteroids.Remove(collision.gameObject);
+                Destroy(collision.gameObject);
+            }
+            else
+            {
+                spawner.SpawnShrapnel(collision.gameObject.transform.position, collision.collider);
```

Comment:
This should be called regardless the health of the asteroid

Reply:
This has been addressed.